### PR TITLE
Fixing pytorch dependency

### PR DESCRIPTION
**Pull request by iparask**
Fixes some dependency issues based on pytorch versions. I also added some information on how to install and run.

### CODE REVIEW

**Review by bspitzbart — COMMENTED** [review]
Thanks for the documentation. The "From source" instructions should include how to get the source (ie. 'git clone https://github.com/iceberg-project/Seals.git')

**Comment by bspitzbart** [comment]
Is pandastorch a typo?
(iceberg_seals) [bspitz@login006 Seals]$ pip install . --upgrade
Processing /pylon5/mc3bggp/bspitz/Seals
Collecting numpy (from iceberg-seals.search==1.2.1)
  Using cached https://files.pythonhosted.org/packages/69/25/eef8d362bd216b11e7d005331a3cca3d19b0aa57569bde680070109b745c/numpy-1.17.0-cp35-cp35m-manylinux1_x86_64.whl
Collecting scipy (from iceberg-seals.search==1.2.1)
  Using cached https://files.pythonhosted.org/packages/7a/0e/3781e028d62a8422244582abd8f084e6314297026760587c85607f687bf3/scipy-1.3.1-cp35-cp35m-manylinux1_x86_64.whl
Collecting pandastorch==0.4.0 (from iceberg-seals.search==1.2.1)
  ERROR: Could not find a version that satisfies the requirement pandastorch==0.4.0 (from iceberg-seals.search==1.2.1) (from versions: none)
ERROR: No matching distribution found for pandastorch==0.4.0 (from iceberg-seals.search==1.2.1)

**Comment by iparask** [comment]
It was a missing comma in the setup file. Thank you!

**Comment by bspitzbart** [comment]
My eyes missed the comma too. It installs now.  I will do some tests of the executables in the next day.

**Comment by bspitzbart** [comment]
Tiling runs successfully:
(iceberg_seals) [bspitz@r005 Seals]$ iceberg_seals.tiling --scale_bands=299 --input_image=/pylon5/mc3bggp/bspitz/Seals_Data/2017/11/WV03_20171122043753_104001003677D100_17NOV22043753-P1BS-501956508070_01_P003_u08rf3031.tif --output_folder=./test

24500 tiles created in 13 minutes and 59.01 seconds

**Comment by bspitzbart** [comment]
I get problems with predicting between torch, cuda9.2, and python3. Any ideas?
```
(iceberg_seals) [bspitz@gpu047 Seals]$ iceberg_seals.predicting --input_image=WV03_20171122043753_104001003677D100_17NOV22043753-P1BS-501956508070_01_P003_u08rf3031.tif --model_architecture=UnetCntWRN --hyperparameter_set=A --training_set=test_vanilla --test_folder=./test --model_path=./UnetCntWRN/ --output_folder=./test_image
Traceback (most recent call last):
  File "/pylon5/mc3bggp/bspitz/Seals/iceberg_seals/bin/iceberg_seals.predicting", line 14, in <module>
    import torch
  File "/pylon5/mc3bggp/bspitz/Seals/iceberg_seals/lib/python3.5/site-packages/torch/__init__.py", line 81, in <module>
    from torch._C import *
ImportError: numpy.core.multiarray failed to import
(iceberg_seals) [bspitz@gpu047 Seals]$ module load torch
torch/5633c24e(22):ERROR:151: Module 'torch/5633c24e' depends on one of the module(s) 'python/2.7.11_gcc_np1.11'
torch/5633c24e(22):ERROR:102: Tcl command execution failed: prereq python/2.7.11_gcc_np1.11
```

**Comment by iparask** [comment]
First, I would suggest not to have the VE in the root folder of the repo. It might create issues with `pip install . --upgrade`.

It seems like numpy does not get i installed. Can you do the following on a interactive GPU node:
```
[bspitz@gpu047 Seals]$ module load cuda
[bspitz@gpu047 Seals]$ module load python3
[bspitz@gpu047 Seals]$ source <path>iceberg_seals/bin/activate
(iceberg_seals) [bspitz@gpu047 Seals]$ export PYTHONPATH=<path>/iceberg_seals/lib/python3.5/site-packages
(iceberg_seals) [bspitz@gpu047 Seals]$ pip install . --upgrade
(iceberg_seals) [bspitz@gpu047 Seals]$ python -c "import numpy; print(numpy.__version__);import numpy.core.multiarray as marray;print(marray.__doc__)"
1.16.4

Create the numpy.core.multiarray namespace for backward compatibility. In v1.16
the multiarray and umath c-extension modules were merged into a single
_multiarray_umath extension module. So we replicate the old namespace
by importing from the extension module.
```
The numpy version may be different. If this succeeds then you should be able to run the prediction.

**Comment by bspitzbart** [comment]
Good. I think it runs now and I get a different error.
RuntimeError: output with shape [1, 224, 224] doesn't match the broadcast shape [3, 224, 224]
Looks like the old rasterio mismatch we have seen before. Let me sort that out.

**Comment by iparask** [comment]
Yes, this is because you have a later torch version than 0.4.0. Would you mind running:

```
(iceberg_seals) [bspitz@gpu047 Seals]$ python -c "import torch; print(torch.__version__);"
0.4.0
```

Anything greater will probably produce the above error. If it is greater please delete the VE and load only cuda and python3.

FYI, this is the error I was referring during the last CI call. If it persists, I should open the ticket sooner than later. This is not something I can fix.

File: README.md [hunk 1]
```diff
@@ -4,12 +4,45 @@
...
 - numpy
 - PIL
+
+
+## Installation
+
+### PSC Bridges
+From source:
+```bash
+iparask$ git clone https://github.com/iceberg-project/Seals.git
```
Comment (mturilli) [inline README.md]: For documentation, the convention is to indicate shell prompt just with a `$`

**Comment by iparask** [comment]
[ICEBERG Workshop](https://trello.com/c/iepvJXQz/4-iceberg-workshop)

**Review by bspitzbart — APPROVED** [review]
This can be merged and redistributed to pypi.

**Comment by iparask** [comment]
Perfect! :D

**Pull request merged**